src/main/java/org/isf/sms/gui/SmsBrowser.java

```diff
@@ -38,12 +38,11 @@
 import org.isf.sms.model.Sms;
 import org.isf.utils.exception.OHServiceException;
 import org.isf.utils.exception.gui.OHServiceExceptionUtil;
+import org.isf.utils.jobjects.CustomJDateChooser;
 import org.isf.utils.jobjects.ModalJFrame;
 import org.joda.time.DateMidnight;
 import org.joda.time.DateTime;
 
-import org.isf.utils.jobjects.CustomJDateChooser;
-
 /**
```

Comment:
not related to this PR, but I'd be in favor of getting rid of these custom @author headers, and maybe replace them with some predefined license/copyright ones (there's even a [maven plugin](https://www.mojohaus.org/license-maven-plugin/) to do that automatically)

Reply:
See this [PR](https://github.com/informatici/openhospital-core/pull/165) which has an example of using the maven plugin.